packages/portal/spec-renderer-mini/src/types/specs.ts

```diff
@@ -10,3 +10,14 @@ export interface SpecItemType {
   selected?: boolean
   key?: string
 }
+
+interface SpecExternalDoc {
+  description: string
+  url: string
```

Comment:
Should there be a `name` attribute to identify the Doc?

Reply:
I'm not sure. I was using the swagger-ui default .yaml as my guide when building the object so it may be only a partial definition and pretty sure it doesn't display a name in the UI because it hides it under some generic "click to see more" text. 

Either way I don't think it matters because we aren't doing anything with that part of the object currently anyway. Perhaps down the line if there is some requirement we can update the definition as needed.

Reply:
OpenAPI [doesn't support](https://swagger.io/specification/#external-documentation-object) external doc `name` by default. It could be extended for our use case, but I don't think it's needed.

FYI _technically_ the `description` field can have CommonMark formatting.